### DISCUSSION TITLE
Licensing information

### DISCUSSION THREAD

**Discussion by MatthieuSarter** [querycap/istio#79] [General] [answered]
Hi @morlay,

I would like to use the Makefile from this repo as a base to start building and publishing istio images for ppc64le, but I found no info about the license covering your code.
Could you please add some details about that ?

Regards,
Matthieu

**Comment by morlay** [comment, marked as answer]
Thank you to tell me. 
Just do anything you want. I added [the Unlicense](https://github.com/querycap/istio/blob/release-1.11/LICENSE)

**Comment by MatthieuSarter** [comment]
Thanks 👍
